### PR TITLE
[CarPlay] Add support for Xcode13 beta1.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/carplay.cs [hunk 1]
```diff
@@ -164,6 +164,97 @@ public enum CPTextButtonStyle : long
...
+		RoundaboutExit10 = 37,
+		RoundaboutExit11 = 38,
+		RoundaboutExit12 = 39,
+		RoundaboutExit13 = 40,
+		RoundaboutExit14 = 41,
+		RoundaboutExit15 = 42,
+		RoundaboutExit16 = 43,
+		RoundaboutExit17 = 44,
+		RoundaboutExit18 = 45,
+		RoundaboutExit19 = 46,
```
Comment (rolfbjarne) [inline src/carplay.cs:248]: I wonder where this roundabout is?

**Review by rolfbjarne — APPROVED** [review]

**Review by spouliot — APPROVED** [review]

File: src/carplay.cs (hunk 1 above)
Comment (dalexsoto) [reply src/carplay.cs:248]: Una rotonda with 19 exits!

**Review by dalexsoto — APPROVED** [review]

File: src/carplay.cs [hunk 2]
```diff
@@ -164,6 +164,97 @@ public enum CPTextButtonStyle : long
...
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPTrafficSide : byte
+	{
+		Right = 0,
+		Left = 1,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPManeuverType : byte
```
Comment (tj-devel709) [inline src/carplay.cs:200]: So the "uint8_t" type should be "byte" in our bindings?

**Review by tj-devel709 — APPROVED** [review]

File: src/carplay.cs (hunk 2 above)
Comment (spouliot) [reply src/carplay.cs:200]: yes, `u` for unsigned and `8` for 8-bits -> `System.Byte`

File: src/carplay.cs (hunk 1 above)
Comment (mandel-macaque) [reply src/carplay.cs:248]: Place Charles de Gaulle is a good contender 


Screenshot 2021-06-14 at 17 47 29

**Pull request merged**